tools/extend_oem_partition.go

```diff
@@ -0,0 +1,71 @@
+package tools
+
+import (
+	"cos-customizer/tools/partutil"
+	"errors"
+	"log"
+	"strconv"
+)
+
+//ExtendOemPartition moves stateful partition towards the end of the disk
```

Comment:
Can we leave a space after `//`? Also, if the sentence is finished, end it with period `.`.